core/services/relay/evm/mercury/v2/reportcodec/report_codec.go

```diff
@@ -31,8 +31,8 @@ func getReportTypes() abi.Arguments {
 		{Name: "feedId", Type: mustNewType("bytes32")},
```

Comment:
According to the PRD, the expected report field ordering here is:
- feedId
- observationTimestamp
- validFromTimestamp
- benchmarkPrice
- linkFee
- nativeFee
- expiresAt